draft-ietf-radext-radiusdtls-bis.md

```diff
@@ -659,6 +659,19 @@ Where the confidentiality of the contents of the RADIUS packet across the whole
 
 [^refto7585]: TODO: Mabe add a reference to handling dynamic discovery (RFC7585) here too, and (as per Alans comments) that this issue is best resolved by limiting use of proxies.
 
+### Loopback-Attack on Peers acting as Server and Client
+
+Radius setups that are configured to act both as client and server, typically in a proxy configuration, SHOULD implement mitigations to avoid accepting connections from itself. Wehen accepting incoming connections from any address (or a wide address range), the certificate subject or subject alternate names cannot be verified, and the trust is based on the certificate issuer or certificate OID. However in this cases, the certificate used for outgoing connections might also satisfy this trust check. Other scenarios where the identification of an outgoing connection satisfies the thrust check of an incoming one are possible, but are not enumerated here.
+
+Either thru misconfiguration, erroneous or spoofed dynamic discovery, or an attacker rerouting TLS packets, a proxy might thus open a connection to itself, creating a loop. Such attacks have been described for TLS-PSK [selfie], dubbed a selfie-attack, but are much broader in the Radius/(D)TLS case. In particular, as descibed above, they also applie to certificate based authentication.
```

Comment:
```suggestion
Either through misconfiguration, erroneous or spoofed dynamic discovery, or an attacker rerouting TLS packets, a proxy might thus open a connection to itself, creating a loop.
Such attacks have been described for TLS-PSK [selfie], dubbed a selfie-attack, but are much broader in the Radius/(D)TLS case. In particular, as described above, they also apply to certificate based authentication.
```

The link to `[selfie]` has not been provided. This should be included in the references.